service/routes.py

```diff
@@ -84,6 +80,58 @@ def create_addresses(customer_id):
 
     return make_response(jsonify(to_send), status.HTTP_201_CREATED)
```

Comment:
shouldnt we return the location url for address too?

Reply:
Okay, Create an address. I would have to take care of it, so it's not a thing for this PR. Do we reopen my closed branch?

Reply:
Addressing this in #44